src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -98,18 +100,22 @@ class IdentityUserAvatar extends React.Component<Props, State> {
         <div>
           {/* Button shape is ButtonShape.Square to make the height and width the same
             so we can use the border radius to make it a circle  */}
-          <Button
-            text={this.getInitials()}
-            shape={ButtonShape.Square}
-            color={
-              isPopoverOpen ? ComponentColor.Default : ComponentColor.Tertiary
-            }
-            onClick={this.togglePopoverState}
-            className={userAvatarButtonClassName}
-          />
+          {this.getInitials() && (
```

Comment:
This is the same adjustment as is made in the user profile avatar PR, to prevent React from trying to render the avatar button before the user's identity information has been fetched (which causes flake).